platform/docs/docs/configuration/theme-configuration.md

```diff
@@ -0,0 +1,238 @@
+# Theme Configuration
+When adding new config-point extendible configuration items, please document
+them here.
+
+## Config-Point
+The configuration schema is based on the [config-point](https://github.com/OHIF/config-point)
+library.  This library is design to allow developers to creation configuration
+points for their code by declaring static values, which can be modified externally.
+See [config-point-service](../platform/services/config-point-service.md) for
+internal details and development documentation.
+
+## Theme Files
+Theme files are simple json5 definition files, placed into the deployment
+under the `https://ohif/theme/` directory.  JSON5 is an extension of JSON,
+for which the primary addition here is the ability to use comments within the
+JSON structure.  The attribute definition can also be an unquoted string when
+it is a simple attribute value.
+
+Their content looks like:
+```js
+{
+  // This extension modifies the ThemeProtocols
+  // by adding a new hanging protocol for MG and modifying the existing
+  // 2x2 layout to make it not preferred.
+  ThemeProtocols: {
+    // The original protocols declaration was a list,
+    // doing this as an object matches by the id value specified here.
+    // With no ID being found, creates a new entry.
+    protocols: {
+      // MG isn't matching, so adds a new item at the end of the list
+      MG: {
+        id: 'MG',
+        // ... rest of definition of hanging protocol
+      },
+      // 2x2 is an existing HP, so modify values rather than replacing/updating it
+      '2x2': {
+        protocolMatchingRules: {
+          // The actual change is weight:5 instead of weight:20, to make this
+          // a non-preferred hanging protocol according to the HP definitions.
+          'NumberOfStudyRelatedSeries>2': { weight: 5 },
+        },
+      }
+    },
+  },
+}
+```
+as an example that modifies the hanging protocols, both by adding and updating
+elements.  Note how this is a deep modification to a value.  The intent is
+to allow modifying configuration values which are heavily nested and/or list
+based.
+
+## Working with Objects
+The object tree is matched by the simple attribute name.  The attribute tree
```

Comment:
I understadn what this section is, but I needed to read twice to figure out that we are explaining how config-point works here (first I thought this is continuation of theme and hp stuff)